gdk4/src/auto/enums.rs

```diff
@@ -264,6 +272,14 @@ impl ToValue for CrossingMode {
     }
 }
 
+impl From<CrossingMode> for glib::Value {
```

Comment:
We don't want these things backported. That's something for the next release. Adding so many trait impls (and also without the gtk-rs-core parts of it!) is dangerous

Reply:
I.e. use the latest gir from the 0.16 branch here, not the master branch

Reply:
> We don't want these things backported. That's something for the next release. Adding so many trait impls (and also without the gtk-rs-core parts of it!) is dangerous

oh i thought that was backported, my bad

Reply:
It needed quite some changes in gstreamer-rs so it continued compiling :)

Reply:
I went the lazy way and added an extra commit for switching to 0.16 branch & regenerating